src/base/bittorrent/session.cpp

```diff
@@ -3929,6 +3929,37 @@ quint64 Session::getAlltimeUL() const
     return m_statistics->getAlltimeUL();
 }
 
+QVariantMap Session::getStats() const
+{
+    const quint64 atd = getAlltimeDL();
+    const quint64 atu = getAlltimeUL();
+
+    // num_peers is not reliable (adds up peers, which didn't even overcome tcp handshake)
+    quint32 peers = 0;
```

Comment:
why are you using `quint32`? `torrent->peersCount()` returns `int`.

Reply:
This is existing code from `StatsDialog::update()`. I'll change it to use an int.